src/LowLevelMotionController.cpp

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// LowLevelMotionController
+//
+// Class to implement control of the quads movement
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include <ros/ros.h>
+#include "QuadVelocityController.hpp"
+
+using namespace Iarc7Motion;
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "Low_Level_Motion_Control");
+
+    ROS_INFO("Low_Level_Motion_Control begin");
+
+    ros::NodeHandle nh;
+
+    QuadVelocityController quadController(nh);
+
+    quadController.init();
+
+    while(ros::ok())
+    {
+        quadController.update();
```

Comment:
I think it's something we should look into if we aren't sure what the timing on this will be, because it would be good to know if there's latency or a low frequency update here because of waiting for transforms